XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
This will walk up folder structure to system root, right? Should it stop at solution root instead?